### PR TITLE
#442 : Adding section

### PR DESCRIPTION
**Pull request by JonasAls89**

### CODE REVIEW

File: training/030_dtl/030_Novice.rst [hunk 1]
```diff
@@ -233,11 +233,60 @@ Basics, uten apply
...
+ - ``"_tracked"`` - if ``true``, the entity was added by `dependency tracking <https://docs.sesam.io/concepts.html#dependency-tracking>`_
+ - ``"_ts"`` - a real-world timestamp for when the entity was added to the dataset
+ - ``"_updated"`` - determines when the entity was modified and the value must either be a string or an integer value
+
+In general, reserved fields are used in order to make Sesam perform as intended and so they will largely affect the performance of a Sesam node without you even knowing. Albeit, you will immediately get to know ``"_deleted"``, ``"_filtered"`` and ``"_id"`` when creating dataflows in Sesam. An entity cannot exist without an ``"_id"``, so it is a given that you will get acquainted with that field immediately. With regards to the ``"_filtered"`` field this allows for you to evaluate and modify whether your DTL logic should transform specific states of an entity and will in general become more apparent as data moves towards exposure in a Sesam dataflow. Finally, the ``"_deleted"`` field is usually used as a filter in your endpoint pipes, as you have to ensure that no deleted entities are exposed to the outside world.
```
Comment (ga-hegsvold) [inline training/030_dtl/030_Novice.rst:256]: If we are to dive into details about these _ properties, I agree that the three you cover here are the most relevant.
Maybe we should also say something about in which situations these properties are encountered.
For example when using ``["filter"]`` you will see `_filtered: true` only in the result view of the pipe preview, and only if the current entity is filtered out.
After running the pipe, the same entity will be `_deleted: true` in the output dataset, but you will **not** see `_filtered` there.

Are there situations where an entity will actually have the `_filtered` property in a dataset?
I can only find it documented under the `filter` function in the DTL Guide.

**Review by ga-hegsvold — CHANGES_REQUESTED** [review]

File: training/030_dtl/030_Novice.rst [hunk 2]
```diff
@@ -233,11 +233,60 @@ Basics, uten apply
...
 
-(_deleted, filtered, \_id, \_previous, \_updated, *\_hash? REF 1.2.24*)
+.. sidebar:: Summary
+
+  ``"_"`` Properties...
+
+  - in Sesam are categorized as `reserved fields <https://docs.sesam.io/entitymodel.html#reserved-fields>`_
+  - provide different functionality for Sesam with regards to how entities are treated
+
+``_`` properties in Sesam are categorized as `reserved fields <https://docs.sesam.io/entitymodel.html#reserved-fields>`_. These fields provide different functionality for Sesam with regards to how entities are treated, as these move through a Sesam dataflow. Only the ``"_id"`` and ``"_deleted"``, will be ignored when writing an entity to a dataset. Additionally, these fields are *only* reserved at the root level, so nested entities can have them.
```
[the remark this answers is not in the document]
Comment (JonasAls89) [reply training/030_dtl/030_Novice.rst:243]: fixed

File: training/030_dtl/030_Novice.rst [hunk 3]
```diff
@@ -233,11 +233,60 @@ Basics, uten apply
...
+
+Below, a complete list of these fields is provided:
```
[the remark this answers is not in the document]
Comment (JonasAls89) [reply training/030_dtl/030_Novice.rst:245]: think it's so brief, and seems to me like a good entry for the section... I'll keep it for now, a bit repetitiveness might consolidate the message.. ;)

File: training/030_dtl/030_Novice.rst [hunk 4]
```diff
@@ -748,11 +748,78 @@ Basics, uten apply
...
+Albeit, you will immediately get to know ``_deleted``, ``_filtered`` and ``_id`` when creating dataflows in Sesam.
+An entity cannot exist without an ``_id``, so it is a given that you will get acquainted with that field immediately.
+
+With regards to the ``_filtered`` field this allows you to evaluate and modify whether your DTL logic should transform specific states of an entity and will in general become more apparent as data moves towards exposure in a Sesam dataflow.
+To exemplify, when using ``["filter"]`` you will see ``{"_filtered": true}`` only in the result view of the pipe preview, and only if the current entity is filtered out.
+After running the pipe, the same entity will be ``{"_deleted": true}`` in the output dataset, but you will not see ``_filtered`` there.
+This is an important aspect to remember to avoid confusion.
+As such, the ``_deleted`` field is added when an entity has been filtered out.
+
+In addition, ``_deleted`` is also used as a filter in your endpoint pipes, as you have to ensure that no deleted entities are exposed to the outside world.
```
Comment (ga-hegsvold) [inline training/030_dtl/030_Novice.rst:788]: The current wording of this sentence implies that you always need to filter out deleted entities in endpoints.
This is sometimes true, but not always. Sometimes the sink type handles this for you. Other times you may want deleted entities to be exported.
I think the sentence should be rephrased a bit to reflect this.

**Review by ga-hegsvold — CHANGES_REQUESTED** [review]

**Review by ga-hegsvold — CHANGES_REQUESTED** [review]
I also think it's worth mentioning that user-defined _ properties can serve as temp properties.

**Pull request merged**